# answer
 return "The block returns 3 in both programs but in the second we 
 	get an error when we try to use x as it was defined within the block
 	and limited to that scope"